yo1k/tic_tac_toe/test/core/test_game.py

```diff
@@ -112,10 +112,10 @@ def test_win_condition(self):
                  Cell(0, 2), False)]
         for cells, cell, expect in args_and_expect_list:
             with self.subTest(expect=expect, cell=cell, cells=cells):
-                actual = Logic._win_condition(Board(cells), cell)
+                actual = Logic.win_condition(Board(cells), cell)  # type: ignore
```

Comment:
What is the reasoning behind making this static method public?

Reply:
As I mentioned before, we can not use protected static method from one module to another, even if it is used inside unittests (both our chekers rise error). And we use this method to test win condition algorithm .